wcfsetup/install/files/lib/data/user/UserAction.class.php

```diff
@@ -628,6 +683,7 @@ public function enable() {
 			'data' => [
 				'activationCode' => 0,
 				'blacklistMatches' => '',
+				'emailConfirmed' => null,
```

Comment:
This should only be part of `enable` if the user does not require a manual activation.